solr/docker/templates/Dockerfile.body.template

```diff
@@ -43,11 +43,8 @@ ARG GITHUB_URL=github.com
 
 RUN set -ex; \
     apt-get update; \
-    apt-get -y install acl dirmngr lsof procps wget netcat gosu tini; \
-    rm -rf /var/lib/apt/lists/*; \
-    cd /usr/local/bin; wget -nv https://${GITHUB_URL}/apangin/jattach/releases/download/v1.5/jattach; chmod 755 jattach; \
```

Comment:
We should be able to move the GITHUB_URL argument from the various places it's referenced